test/test-tail.js

```diff
@@ -0,0 +1,19 @@
+var Lab = require('lab');
+var lab = exports.lab = Lab.script();
+
+var describe = lab.describe;
+var it = lab.it;
+var Code = require('code');
+var expect = Code.expect;
+
+var tail = require('../tail');
+
+describe('tail', function () {
+  it('should return the tail from an array, string, or object', function(done) {
+    expect(tail([1, 2, 3])).to.deep.equal([2, 3]);
+    expect(tail('123')).to.equal('23');
+    expect(tail(function (a) {})).to.equal('unction (a) {}');
```

Comment:
I like `tail` function but not sure about automatic conversion to the string. This may introduce problems that would be hard to find in the client code.